src/main/java/uk/gov/hmcts/reform/sscs/idam/UserRole.java

```diff
@@ -1,21 +1,19 @@
 package uk.gov.hmcts.reform.sscs.idam;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
 public enum UserRole {
-    TCW("caseworker-sscs-registrar"),
-    CTSC_CLERK("caseworker-sscs-clerk"),
-    DWP("caseworker-sscs-dwpresponsewriter"),
-    JUDGE("caseworker-sscs-judge"),
-    SUPER_USER("caseworker-sscs-superuser"),
-    SYSTEM_USER("caseworker-sscs-systemupdate"),
-    CITIZEN("citizen");
+    SUPER_USER("caseworker-sscs-superuser", "Super User"),
```

Comment:
Have these changes been tested fully, always a bit cautious where user role related changes, any knock effects in calls to retrieve the user role anywhere in tribunals?

Reply:
All the values are the same so they should be the same, its just extending them to have labels as well